agent/eni/pause/pause_linux.go

```diff
@@ -48,7 +49,9 @@ func loadFromFile(path string, dockerClient engine.DockerClient, fs os.FileSyste
 		return errors.Wrapf(err,
 			"pause container load: failed to read pause container image: %s", path)
 	}
-	if err := dockerClient.LoadImage(pauseContainerReader, engine.LoadImageTimeout); err != nil {
+	ctx, cancel := context.WithTimeout(context.TODO(), engine.LoadImageTimeout)
```

Comment:
You shouldn't need to create a new context here, using `context.TODO()`. You should instead wire it from the "app" package (via loader. LoadImage)

Reply:
ok, sounds good.